README.md

```diff
@@ -19,10 +19,14 @@ We've turned on [discussions](https://github.com/UCL-ARC/python-tooling/discussi
 
 ## Using this template
 
-1. Install [cookieninja]
-   ```
-   pip install cookieninja
-   ```
+1.  Install [cookieninja](https://libraries.io/pypi/cookieninja) in a `conda`, `mamba` or `venv` environment (commented lines for conda example).
```

Comment:
Should we recommend people install `cookieninja` using [`pipx`](https://pypa.github.io/pipx/) - this way they don't have tocreate an environment just for it (or add it to their project environment)

Reply:
I think we should stick with the initial `pip install cookieninja`. For those that know, they'll know what to do anyway. Also, `conda` and `mamba` are the same.